Common/MVKOSExtensions.mm

```diff
@@ -134,7 +134,11 @@ uint64_t mvkGetUsedMemorySize() {
 	task_vm_info_data_t task_vm_info;
 	mach_msg_type_number_t task_size = TASK_VM_INFO_COUNT;
 	if (task_info(mach_task_self(), TASK_VM_INFO, (task_info_t)&task_vm_info, &task_size) == KERN_SUCCESS) {
+#ifdef TASK_VM_INFO_REV3_COUNT	// check for rev3 version of task_vm_info
+		return task_vm_info.ledger_tag_graphics_footprint;
```

Comment:
What happens if MoltenVK is built with an SDK that has `REV3` struct, but run on a system without it?

Reply:
Good catch.  I wonder if this can be tested at runtime?  Let me look into it.  I also just pushed a small defensive guard change.  I may need one more to address your question.

Reply:
I submitted a change which hopefully solves your runtime question.  Let me know...